lfucg-exactions/client/js/components/SubdivisionSummary.js

```diff
@@ -40,98 +40,103 @@ class SubdivisionSummary extends React.Component {
         
         return (
             <div className="subdivision-summary">
-                <Navbar />
+                {!!subdivisions && !!subdivisions.currentSubdivision && <div>
```

Comment:
@KellyInnovation Just thought I would point out something that Matt showed me with stuff like this... Destructuring. Basically giving react something as initial state to go off of to keep it from breaking
In the this.props declaration I believe you can give subdivisions an initial value like `subdivisions = []` and then below that you can do something like `cont { currentSubdivision = {} } = subdivision` and just call currentSubdivision down in the body as needed with no dot syntax. Thought you'd find it interesting. Keeps you from having to do all the && logic and what not. Maybe you knew that already though.